### PR TITLE
Add Prometheus monitoring settings topic

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-62792

## Description of changes being made
Added to Konvoy docs a topic outlining a table with recommended monitoring setting for Prometheus 

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by russurquhart1** [comment]
https://jira.d2iq.com/browse/D2IQ-62792

File: pages/ksphere/konvoy/latest/monitoring/recommendations/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+layout: layout.pug
+navigationTitle: Recommendations
+title: Recommendations
+menuWeight: 11
+excerpt: Recommended settings for monitoring and collecting metrics for Kubernetes, platform services, and applications deployed on the Konvoy cluster
+---
+
+
+This table provides recommended settings for certain addons, based on cluster size and expected capacity, in terms of pods, that satisfy the needs and workloads of our mixed workload testing (MWT)tests.
```
Comment (samvantran) [inline pages/ksphere/konvoy/latest/monitoring/recommendations/index.md]:
```suggestion
At D2iQ, we conduct routine performance testing of Konvoy. The following table provides recommended settings, based on cluster size and increasing workloads, that maintain a healthy Prometheus monitoring deployment.
```

File: pages/ksphere/konvoy/latest/monitoring/recommendations/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>The resource settings reflect some settings but do not represent the exact structure to be used in the Konvoy `cluster.yaml`.</p>
```
Comment (samvantran) [inline pages/ksphere/konvoy/latest/monitoring/recommendations/index.md]: Pretty sure the `cluster.yaml` doesn't render on the site when inside `<strong>` tags. I think you have to wrap it in `<code>` html tags

**Comment by russurquhart1** [comment]
Fixed Sam's comments in this last commit.

**Pull request merged**